e2e/ansible/playbooks/utils/access-mode-check.yml

```diff
@@ -37,6 +50,32 @@
          delay: 30
          retries: 5
 
+       - name: Get the mayactl version
+         shell: source ~/.profile; kubectl exec -it {{ mayapod.stdout }} -n openebs -- mayactl version | grep -i Version | awk 'FNR == 1 {print}' | awk {'print $2'}
+         args:
+           executable: /bin/bash
+         register: mayactl_version
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         changed_when: True
+
+       - name: Set Mayactl version to a variable
+         set_fact:
+           mayactl: "{{mayactl_version.stdout.split('-')[0]}}"
+
+       - name: Get the replicas access mode
+         shell: >
+            source ~/.profile;  
+            kubectl exec -it {{ mayapod.stdout }} -n openebs -- mayactl volume describe --volname "{{volname.stdout_lines[2].split()[1]}}" -n {{ ns }} | grep 'RW' | wc -l
+         args:
+           executable: /bin/bash
+         register: result
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         until: result.stdout| int == replica_count.stdout | int
+         delay: 30
+         retries: 15
+         changed_when: True
+         when: "mayactl is version('0.7.0', '>')"
```

Comment:
Considering describe is used in & also the check is for > 0.7.0, is there a plan to have an alternate task for =0.7.0 (in case of runs against some patches) ? Should we have another task with older mayactl/curl command for this ?

If not, do we need a version check at all ?

Reply:
:+1:  on using ansible version compare test/filter

Reply:
@ksatchit in the next task itself checking the condition for <= 0.7.0 version

Reply:
Thx !